### PR TITLE
Homebrew 2.5.0

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Add post for Homebrew 2.5.0 release.

### CODE REVIEW

File: _posts/2020-09-04-homebrew-2.5.0.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Major changes and deprecations since 2.4.0:
+
+- Many `brew cask` [commands](https://github.com/Homebrew/brew/pull/8387) [have](https://github.com/Homebrew/brew/pull/8302) - [been](https://github.com/Homebrew/brew/pull/8387) (@MLH-Fellowship)
+ replaced with `brew` commands and are deprecated. This continues our goal of better integration between `brew` and `brew cask` commands.
+- [Most Homebrew/homebrew-core formulae have their licenses stored and audited](https://github.com/Homebrew/brew/pull/7762).
+- [`brew typecheck` provides the beginnings of Homebrew's use of Sorbet for type-checking](https://github.com/Homebrew/brew/pull/8289).
+- [Homebrew is currently experimenting with GitHub Discussions as a possible replacement for Discourse](https://github.com/Homebrew/brew/pull/8570).
+- [`brew livecheck` has moved from its own tap to a Homebrew command and part of Homebrew/homebrew-core formulae](https://github.com/Homebrew/brew/pull/8180).
+- [`brew bump` is a new command using `brew livecheck` and Repology output to outdated formualae in Homebrew/homebrew-core](https://github.com/Homebrew/brew/pull/7834).
```
Comment (issyl0) [inline _posts/2020-09-04-homebrew-2.5.0.md:15]: There's a missing word here between "output to" and "outdated formulae", but I'm bleary eyed so I don't know what it should be. Maybe "check for"?

File: _posts/2020-09-04-homebrew-2.5.0.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: 2.5.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 2.5.0. The most significant changes since 2.4.0 are better `brew cask` integration, license support and API deprecations.
+
+Major changes and deprecations since 2.4.0:
+
+- Many `brew cask` [commands](https://github.com/Homebrew/brew/pull/8387) [have](https://github.com/Homebrew/brew/pull/8302) - [been](https://github.com/Homebrew/brew/pull/8387) (@MLH-Fellowship)
```
Comment (issyl0) [inline _posts/2020-09-04-homebrew-2.5.0.md:9]: None of the other PRs linked have their author specified.

**Review by issyl0 — CHANGES_REQUESTED** [review]

File: _posts/2020-09-04-homebrew-2.5.0.md (hunk 2 above)
Comment (jonchang) [inline _posts/2020-09-04-homebrew-2.5.0.md:9]:
```suggestion
- Many `brew cask` [commands](https://github.com/Homebrew/brew/pull/8387) [have](https://github.com/Homebrew/brew/pull/8302) [been](https://github.com/Homebrew/brew/pull/8387)
```

File: _posts/2020-09-04-homebrew-2.5.0.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- [Homebrew on Linux is moving towards a pure-Ruby ELF reading and writing implementation](https://github.com/Homebrew/brew/pull/7213).
+- [Various other Homebrew (mostly internal) APIs have been deprecated](https://github.com/Homebrew/brew/pull/8513)
+
+Other changes since 2.4.0 I'd like to highlight are the following:
+
+- [`brew sh` has a non-interactive mode](https://github.com/Homebrew/brew/pull/8482).
+- [Disabled and deprecated formulae require a reason (to better explain to users)](https://github.com/Homebrew/brew/pull/8512).
+- [`brew bump-revision` accepts multiple formulae](https://github.com/Homebrew/brew/pull/8358).
+- [`brew cask --help` output is supported for all `brew cask` commands](https://github.com/Homebrew/brew/pull/8239).
+- [audit: add --tap argument.](https://github.com/Homebrew/brew/pull/8231).
```
Comment (jonchang) [inline _posts/2020-09-04-homebrew-2.5.0.md:25]:
```suggestion
- [`brew audit` now has a `--tap` argument](https://github.com/Homebrew/brew/pull/8231).
```

File: _posts/2020-09-04-homebrew-2.5.0.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- [`brew tap-new` makes use of Homebrew's custom GitHub Actions (which are also used by Homebrew CI)](https://github.com/Homebrew/brew/pull/8210).
+- [`brew audit` is passing and mandatory for all changes on Homebrew/homebrew-core](https://github.com/Homebrew/brew/pull/8153).
+- [`brew update-python-resources` is a new command that can be used to upgrade resources on Python formulae](https://github.com/Homebrew/brew/pull/8059).
```
Comment (jonchang) [inline _posts/2020-09-04-homebrew-2.5.0.md:28]:
```suggestion
- [`brew update-python-resources` is a new command that can be used to upgrade dependencies in Python formulae](https://github.com/Homebrew/brew/pull/8059).
```

File: _posts/2020-09-04-homebrew-2.5.0.md (hunk 1 above)
Comment (MikeMcQuaid) [reply _posts/2020-09-04-homebrew-2.5.0.md:15]:
```suggestion
- [`brew bump` is a new command using `brew livecheck` and Repology to output outdated formualae in Homebrew/homebrew-core](https://github.com/Homebrew/brew/pull/7834).
```

**Comment by MikeMcQuaid** [comment]
Thanks for review @jonchang and @issyl0!

**Pull request merged**